#! /bin/sh /usr/share/dpatch/dpatch-run
## skinsbugfix.dpatch by Michael Eiler <eiler.mike@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch for the newplugin-script, to not require a normal vdr-source-dir

@DPATCH@
--- vdr/skins.c.orig    2011-06-26 22:34:44.307839991 +0200
+++ vdr/skins.c 2011-06-26 22:35:23.407840004 +0200
@@ -248,7 +248,8 @@
         cStatus::MsgOsdClear();
         }
      else {
-        cSkinDisplay::Current()->SetMessage(Type, NULL);
+        if (cSkinDisplay::Current())
+           cSkinDisplay::Current()->SetMessage(Type, NULL);
         cStatus::MsgOsdStatusMessage(NULL);
         }
      }
